src/main/java/net/pod/peaengine/gameloop/GameLoop.java

```diff
@@ -20,6 +20,19 @@ public static void setLoopType(Class<? extends GameLoopExecutor> type) {
         GameLoop.type = type;
     }
 
+    /**
+     * Gets current running {@link GameLoopExecutor}
+     * @throws NullPointerException if executor is null
+     * @return current running {@link GameLoopExecutor}
+     */
+    public static GameLoopExecutor getExecutor() {
+        if (executor == null) {
+            throw new NullPointerException("There is no executor running!");
```

Comment:
The exception used is not the correct type, should be IllegalStateException instead

Reply:
Oh oop